assets/js/base/components/product-order-select/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import OrderSelect from '../order-select';
+
+const ProductOrderSelect = ( { defaultValue, onChange, readOnly, value } ) => {
+	return (
+		<OrderSelect
+			className="wc-block-product-order-select"
+			defaultValue={ defaultValue }
+			onChange={ onChange }
+			options={ [
```

Comment:
Suggest taking the options from woo core since they already exist in the API.

Reply:
In the API they are a combination of `order` and `orderby`, right? How would you use those values in a `<select>`? I imagine we could use `price-desc` and `price-asc` as option keys, but I think UI components should be completely independent of the API. So if, for example, some day the API attribute was to be renamed from `price` to `value`, we should not need to update the UI components.

Reply:
I mean if we use what's currently in woo core on the shop page, we know those options are covered by the API.

Reply:
Oh, ok. You mean having those options, no?
<img src="https://user-images.githubusercontent.com/3616980/64117635-fb36bb00-cd95-11e9-96d4-a0844d7cabf1.png" alt="" width="148" />

Reply:
Yup those although I'd love to replace default with something new at some point :)

Reply:
Updated in aa5f996bf2b84faebddb255acac31ac411d1602c.